templates/etc/init.d/rpm/td-agent

```diff
@@ -209,7 +210,15 @@ do_start() {
   fi
 
   local RETVAL=0
-  daemon --pidfile="${TD_AGENT_PID_FILE}" ${START_STOP_DAEMON_ARGS} "${TD_AGENT_RUBY}" ${TD_AGENT_ARGS} || RETVAL="$?"
+  # daemon uses the return code of the underlying bash process as its return
+  # code. So, if you background the first command, the return code will always
```

Comment:
How about s/`you`/`we just`/, to put this comment in the context of the script, rather than our previous discussion?